data/provider/src/main/java/tech/pegasys/teku/api/ChainDataProvider.java

```diff
@@ -562,6 +559,7 @@ public void requireStoreAvailable() {
       final tech.pegasys.teku.datastructures.state.BeaconState state) {
     return validatorIndices.stream()
         .map(index -> ValidatorResponse.fromState(state, index))
+        .flatMap(Optional::stream)
```

Comment:
nit: this one can just use `.flatMap` instead of `.map` as it's not an `IntStream`.

Reply:
im doing something wrong, because i the compiler didnt like me on this one...